packages/utilities/src/array.ts

```diff
@@ -15,7 +15,9 @@ export function createArray(size: number, getItem?: (index?: number) => any) {
   let array = [];
 
   for (let i = 0; i < size; i++) {
-    array.push(getItem(i));
+    if (getItem) {
```

Comment:
if you're going to add an if check, might as well do it outside the for loop once.